kvmagent/kvmagent/test/unittest_tools/prepare_env.sh

```
@@ -174,8 +184,17 @@ copy_kvm_virtualenv_to_venv3() {
     source /root/venv3/bin/activate
+
+    # build pypi source in /root/.zguest/zstack-utility/kvmagent/kvmagent/test/unittest_tools/unittest_pypi_source/pypi
     cd /root/.zguest/zstack-utility/kvmagent/kvmagent/test/unittest_tools/unittest_pypi_source/
+    mkdir -p ./requirements
+    mkdir -p ./pypi
+    bash pypi_source_builder.sh build \
+        --package-list ./package_list_base.txt --target-path ./pypi --requirement-output-path ./requirements/requirements1.txt
+    bash pypi_source_builder.sh build \
+        --package-list ./package_list_ut.txt --target-path ./pypi --requirement-output-path ./requirements/requirements3.txt
+
     pip install -r requirements/requirements1.txt -i file://`pwd`/pypi/simple
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)